README.md

```diff
@@ -17,30 +17,32 @@ To launch a Talk server of your own from your browser without any need to muck a
 
 ### Configuration
 
-The Talk application requires specific configuration options to be available
-inside the environment in order to run, those variables are listed here:
+The Talk application looks for the following configuration values either as
```

Comment:
This is not the case. We should not advertise in the "Configuration" section of the README that the application runs with a `.env` file because this is a development option only, and does not follow best practices for loading the configuration into the environment. We should only include these details in the developer documentation rather than the standard docs where I would assume someone would look if they are trying to deploy the application.

Additionally, the configuration option `-c` will load **any specified file** rather than specifically a `.env` file. The `.env` file only loads with the `dev-start` npm script.